### PR TITLE
[AS-1009] Add metrics for different menu items accessible through entities table

### PR DESCRIPTION
**Pull request by jordansafer**
Added metrics for entity tables in the Workspace Data page. New metrics for: Delete selected rows, Export rows to another workspace, and Open with: IGV, Workflow, Notebook, and Data Explorer
Updated metric for downloading selected rows as a TSV, so we can differentiate from users selecting the Download All Rows button

Testing: Tested all operations work as expected after the change. Verified the new metrics are populated in MixPanel for terra dev.

### CODE REVIEW

File: src/components/data/EntitiesContent.js [hunk 1]
```diff
@@ -146,7 +149,12 @@ const ToolDrawer = _.flow(
               text: 'Workflow'
             }),
             h(ModalToolButton, {
-              onClick: !openDataExplorerInSameTab ? onDismiss : undefined,
+              onClick: () => {
+                Ajax().Metrics.captureEvent(Events.workspaceDataOpenWithDataExplorer, {
+                  ...extractWorkspaceDetails(workspace.workspace)
+                })
+                if (!openDataExplorerInSameTab) onDismiss()
```
Comment (petesantos) [inline src/components/data/EntitiesContent.js]: nit: we typically use short circuiting for these changes
```suggestion
                !openDataExplorerInSameTab && onDismiss()
```

**Review by petesantos — APPROVED** [review]
Looks good - one minor nit

File: src/components/data/EntitiesContent.js (hunk 1 above)
Comment (jordansafer) [reply src/components/data/EntitiesContent.js]: Fixing!

File: src/components/data/EntitiesContent.js [hunk 2]
```diff
@@ -391,6 +399,9 @@ const EntitiesContent = ({
           setDeletingEntities(false)
           setSelectedEntities({})
           setRefreshKey(_.add(1))
+          Ajax().Metrics.captureEvent(Events.workspaceDataDelete, {
+            ...extractWorkspaceDetails(workspace.workspace)
+          })
```
Comment (sagehen03) [inline src/components/data/EntitiesContent.js]: I don't think it's necessary to use the spread operator to create a new object.
```suggestion
          Ajax().Metrics.captureEvent(Events.workspaceDataDelete, extractWorkspaceDetails(workspace.workspace))
```

**Review by sagehen03 — APPROVED** [review]
I don't see any bugs but I think we may be creating extra objects to pass to the payload for captureEvent.

File: src/components/data/EntitiesContent.js (hunk 2 above)
Comment (jordansafer) [reply src/components/data/EntitiesContent.js]: fixing!

**Review by ansingh7115 — APPROVED** [review]

**Pull request merged**